src/common/is-result-highlight-unavailable.ts

```diff
@@ -25,17 +22,22 @@ export const getUnavailableHighlightStatusUnified: GetUnavailableHighlightStatus
         result.descriptors.boundingRectangle == null ||
         !hasValidBoundingRectangle(result.descriptors.boundingRectangle, platformInfo.viewPortInfo)
     ) {
-        return 'unavailable';
+        return true;
     }
 
-    return null;
+    return false;
 };
 
 function hasValidBoundingRectangle(
```

Comment:
Do we want to be showing rectangles if they are only partially within the viewport? Right now this function is saying "yes" for rectangles that are partially occluded to the right/bottom but "no" for rectangles partially occluded to the left/top; feels like the answer should be consistent

Reply:
I was thinking about that as well: based off of previous discussions with Fer, it seemed like we were fine with visualizations that were at least partially in-frame going to the right. I guess that would make it inconsistent here ... let me check with Fer to get an explicit condition we're comfortable with moving forward.